flow/aggregates.py

```diff
@@ -432,8 +421,5 @@ def get_aggregate_id(jobs):
         return jobs[0].get_id()  # Return job id as it's already unique
 
     blob = ','.join((job.get_id() for job in jobs))
-    return f'agg-{_hash(blob)}'
-
-
-def _hash(blob):
-    return md5(blob.encode('utf-8')).hexdigest()
+    hash_ = md5(blob.encode('utf-8')).hexdigest()
+    return f'agg-{hash_}'
```

Comment:
The separate function was a little bit overkill -- it's only used once.

Reply:
My original intent intent in requesting a separate function was to also include the logic for blob construction (which was being done in many places in this file), but as you've pointed out you can always just hash a tuple of things rather than build up that object which is better :)